syntax = "proto3";

package protobuff;

message PersonTypePb {
    PersonTypeEnum personType = 1;
}

enum PersonTypeEnum{
    UNKNOWN_PERSON_TYPE=0;
    WORKER=1;
    CONSUMER=2;
}
